kivy/lang/parser.py

```diff
@@ -422,6 +422,10 @@ def execute_directives(self):
                     ref = ref[6:].strip()
                     force_load = True
 
+                # if #:include [force] "path with quotes around"
+                if ref[0] == ref[-1] and ref[0] in ('"', "'"):
+                    ref = ref[1:-1]
```

Comment:
well, exactly that's why the three alternatives i.e. the second one (ccb1f7d) doesn't care about how many of them you have and just strips all, but anyway, triple quotes rather make sense when making strings multiline which isn't possible with #:include as the parser goes line-by-line so unless we'd go for a completely new way of handling directives, I don't see it hapenning otherwise than just this:

    #:include [force] """C:\my path"""

and slicing them off. That seems rather ugly though.